independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/project/compress/QuarkusProjectCompress.java

```diff
@@ -19,9 +19,7 @@
 
     private static final List<String> EXECUTABLES = Collections.unmodifiableList(Arrays.asList(
```

Comment:
I think we can also use JDK 11's `List.of` here instead

Reply:
Thanks. Fixed